src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,13 @@
+export default function ArticleCard({ article, index }) {
+  return (
+    <li className="article" id={index}>
+      <img src={article.article_img_url} alt="article image" />
```

Comment:
The alt tag here isn't great, now I know that you're mapping over a lot of data and this is generated for each one. Is there anything attached to the article that you could use to make a dynamic alt tag?

Reply:
As these images are decorative, it would be better to leave the alt tag as an empty string in this case as there isn't really a great way to make a meaningful alt tag per image.